### PR TITLE
Panorama fix

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22465

## Description
add checks for None in some places.

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [ ] Documentation

### CODE REVIEW

**Comment by kirbles19** [comment]
@esharf Done. Please review for technical accuracy

File: Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -653,25 +653,27 @@ def panorama_push_status_command():
...
-    push_status_output = {'JobID': result['response']['result']['job']['id']}
-    if result['response']['result']['job']['status'] == 'FIN':
-        if result['response']['result']['job']['result'] == 'OK':
+    push_status_output = {'JobID': job.get('id')}
+    if job.get('status', '') == 'FIN':
+        if job.get('result', '') == 'OK':
             push_status_output['Status'] = 'Completed'
         else:
-            # result['response']['job']['result'] == 'FAIL'
+            # job['result'] == 'FAIL'
```
Comment (yaakovi) [inline Integrations/Panorama/Panorama.py]: why is this in comment?

@bakatzir can you give your take here?

File: Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
 class PAN_OS_Not_Found(Exception):
     """ PAN-OS Error. """
-    def __init__(self, *args, **kwargs):  # real signature unknown
+    def __init__(self, *args):  # real signature unknown
```
Comment (yaakovi) [inline Integrations/Panorama/Panorama.py]: why did you remove the kwargs?

Comment (bakatzir) [reply Integrations/Panorama/Panorama.py]: it is not needed

File: Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (bakatzir) [reply Integrations/Panorama/Panorama.py]: To simplify for the reviewer, can be removed.
